test/lib/util/fast_json_encoder_test.exs

```diff
@@ -0,0 +1,61 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.FastJasonEncoderTest do
+  use Croma.TestCase
+  use ExUnitProperties
+
+  def validate(value) do
+    validate(value, value)
+  end
+  def validate(source, expected) do
+    {:ok, json} = FastJasonEncoder.encode(source)
+    assert json == Poison.encode!(source)
+    assert expected == Poison.decode!(json)
+  end
+
+  test "primitives" do
+    values = [0, 1, 1.5, true, false, nil]
+    Enum.each(values, &validate(&1))
```

Comment:
Other parts are the same, too.
```suggestion
    Enum.each(values, &validate/1)
```

Reply:
Fixed b0c7c48381997e5ac476ebf9b332e49eab348fd4